src/biogeochem/CNBalanceCheckMod.F90

```diff
@@ -319,20 +326,42 @@ subroutine NBalanceCheck(this, bounds, num_soilc, filter_soilc, &
          ! calculate the total column-level nitrogen balance error for this time step
          col_errnb(c) = (col_ninputs(c) - col_noutputs(c))*dt - &
               (col_endnb(c) - col_begnb(c))
-
-         if (abs(col_errnb(c)) > 1e-3_r8) then
-            err_found = .true.
-            err_index = c
-         end if
+        
+!        if(c .eq. 1411 .or. c .eq. 2677)then
+!           write(*,*)'column nbalance error    = ',col_errnb(c), c
+!           write(*,*)'Latdeg,Londeg            = ',grc%latdeg(col%gridcell(c)),grc%londeg(col%gridcell(c))
+!           write(*,*)'begnb                    = ',col_begnb(c)
+!           write(*,*)'endnb                    = ',col_endnb(c)
+!           write(*,*)'delta store              = ',col_endnb(c)-col_begnb(c)
+!           write(*,*)'input mass               = ',col_ninputs(c)*dt
+!           write(*,*)'output mass              = ',col_noutputs(c)*dt
+!           write(*,*)'net flux                 = ',(col_ninputs(c)-col_noutputs(c))*dt
+!           write(*,*)'inputs,ffix,nfix,ndep    = ',ffix_to_sminn(c)*dt,nfix_to_sminn(c)*dt,ndep_to_sminn(c)*dt
+!           write(*,*)'outputs,ffix,nfix,ndep   = ',smin_no3_leached(c)*dt, smin_no3_runoff(c)*dt,f_n2o_nit(c)*dt
+!        end if
          
-         if (abs(col_errnb(c)) > 1e-7_r8) then
-            write(iulog,*) 'nbalance warning',c,col_errnb(c),col_endnb(c)
-            write(iulog,*)'inputs,ffix,nfix,ndep = ',ffix_to_sminn(c)*dt,nfix_to_sminn(c)*dt,ndep_to_sminn(c)*dt
-            write(iulog,*)'outputs,lch,roff,dnit = ',smin_no3_leached(c)*dt, smin_no3_runoff(c)*dt,f_n2o_nit(c)*dt
-         end if
+        if(use_matrixcn .or. use_soil_matrixcn)then 
+           if (abs(col_errnb(c)) > 1.e-1_r8) then   !1e-3_r8
```

Comment:
Please remove the comment in this line. Or is it intentionally written there?

Reply:
The comment has been removed.